server/config/disaster_dashboard/Earth.textproto

```diff
@@ -274,12 +276,14 @@ categories {
           event_type_keys: "fire"
         }
       }
-      # tiles {
-      #   # TODO: Need to specify P1Y
-      #   type: HISTOGRAM
-      #   title: "Number of fires last year"
-      #   stat_var_key: "fire_count"
-      # }
+      tiles {
+        # TODO: Need to specify P1Y
```

Comment:
you can delete this comment if it doesn't apply anymore (ditto for the rest)

Reply:
Done!